.travis.yml

```diff
@@ -45,10 +45,37 @@ matrix:
       script: mvn test-compile -Pall-java,licenses,lgpl,checkstyle,examples,check-licenses,!spark-2.4,!spark,!scala -Dmaven.javadoc.skip=true -B -V
       before_cache: rm -rfv $HOME/.m2/repository/org/apache/ignite
       cache: {directories: [$HOME/.m2/repository]}
-
+      
+    - language: java
+      os: linux
+      arch: s390x
```

Comment:
Travis build is meant as a quick check for Pull Requests:
* Code compiles under JDK 8 and 11
* Code style is ok

s390x may be useful, but it should be added to our TeamCity instead:
https://ci.ignite.apache.org/

Reply:
@ptupitsyn Does TeamCity support s390x?
If yes, can you please suggest steps how s390x support can be added to TeamCity?

Reply:
I don't know, probably a new agent will be required. Please ask about this on [dev list](dev@ignite.apache.org).

Reply:
@ptupitsyn I have mailed on dev list.